repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Authorization.scala

```diff
@@ -59,13 +56,24 @@ object KeystoneV2Authorization extends LazyLogging {
     }
   }
 
-  val handleFailures: PartialFunction[Try[Unit.type], KeystoneV2Result] = {
-    case Failure(e: InvalidTenantException) => Reject(SC_UNAUTHORIZED, Some(e.getMessage))
-    case Failure(e: UnauthorizedEndpointException) => Reject(SC_FORBIDDEN, Some(e.getMessage))
-    case Failure(e: UnparseableTenantException) => Reject(SC_UNAUTHORIZED, Some(e.getMessage))
+  // TODO: Switch to getRequestTenants (plural) and include all tenants extracted. This is a behavior change.
```

Comment:
So as written, this code will look for a tenant in a header as well as in the URI, giving preference to the value in a header. However, it only returns a single tenant ID. This is consistent with current behavior given multiple URI extraction regexes, but is going to change with the support for multi-tenants story. I just wanted to make a note of that.

Reply:
Pretty sure this comment is out of date

Reply:
Not in this code, which does not yet support the multi-tenant use-case. The next story handles it, and removes the TODO.

Reply:
I meant your pr comment, not the code comment.